lib/lita/extensions/confirmation.rb

```diff
@@ -24,6 +24,7 @@ def call
           message.reply(
             I18n.t(
               "lita.extensions.confirmation.request",
+              prefix: @robot.alias ? @robot.alias : "#{@robot.mention_name} ",
```

Comment:
`robot.mention_format(robot.mention_name)` would be better, to ensure the right format is used for whatever adapter is being used. This will require bumping the dependency on Lita to >= 4.4.0, which is fine.